sharing-service/services/sharing/dynamoDBupdateScript.js

```diff
@@ -0,0 +1,16 @@
+
+  const result = await dynamoDocClient()
```

Comment:
This file is only for testing, I guess. It should be somewhere else, then, like the way checklist-service has `scripts/dynamodb`